misc/python/materialize/cli/scratch/push.py

```diff
@@ -0,0 +1,36 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import argparse
+
+import boto3
+
+from materialize.cli.scratch import check_required_vars
+from materialize.scratch import mkrepo
+
+
+def configure_parser(parser: argparse.ArgumentParser) -> None:
+    check_required_vars()
+
+    parser.add_argument("instance", help="The ID of the instance to connect to"),
+    parser.add_argument("--rev", help="The git rev to checkout", default="HEAD"),
+    parser.add_argument(
+        "--force",
+        help=(
+            "Whether to force push "
+            "(useful to keep your target/ directory as you are changing tests)"
+        ),
+        action="store_true",
+    ),
```

Comment:
I feel like we should just always `--force`? The `refs/heads/scratch` refspec is meant to be something that the `bin/scratch` script owns, rather than a branch whose history is important.

Reply:
I am happy with always forcing!